src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdServerSettings.Edit.cshtml

```diff
@@ -2,6 +2,7 @@
 @using OrchardCore.OpenId.Settings;
 @using System.Security.Cryptography.X509Certificates
 @model OpenIdServerSettingsViewModel
+@inject Microsoft.Extensions.Options.IOptions<OpenIddict.Server.OpenIddictServerOptions> OpenIddictServerOptions
```

Comment:
Same remark as above, this won't do what you think it does (basically, you'll only get uninitialized options with empty paths).